nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -42,25 +42,18 @@ def go_to_patient(self):
         # Assume door is open
         # TODO: first go to doorside then bedside when door opening added
         status = self.navigate_task.execute(self.room, "bedside")
-        if status == Task.ERROR:
-            raise NotImplementedError()
         self.phases.put(self.triage_patient)
 
     def go_home(self):
         rospy.loginfo("Nightingale Mission Planner going home")
         # Assume door is open
         # TODO: first go to doorside then bedside when door opening added
         status = self.navigate_task.execute("home", "")
-        if status == Task.ERROR:
```

Comment:
keep in mind that there are currently 2 different things that are "Not Implemented". One is the execute method for some classes. The other is the behavior in case the task return FAILURE. From a quick glance at this PR it looks like ur combining both?

Reply:
OHHHH I completly missunderstood this, I didn't recognize that raise not implemented was being used as stubs for error handeling as well. I was assuming we were completly ignoring error checking for V1. My PR breaks your original intention for error handeling so I think this shouldn't get merged